static/script/devices/mediaplayer/mediaplayer.js

```diff
@@ -154,14 +154,13 @@ require.def(
             },
 
             /**
-             * Request that the media start playing from Time.
+             * Request that the media seeks to Time.
```

Comment:
For clarity, can this say 'Reques that the media seeks to and starts playing from Time'   :-)